import{_ as b,A as y,a as v}from"./index.6780d7bf.js";import{E as w}from"./element-plus.7b496d94.js";import{ag as i,o as x,c as B,X as s,R as o,V as n,a as e,aw as S,ax as D}from"./@vue.88ba4ddc.js";import"./geotiff.b2c33c96.js";import"./vue-router.1c4533a7.js";import"./ol.c3d2637f.js";import"./rbush.314566ea.js";import"./quickselect.bc77a5ac.js";import"./@element-plus.eabea6f1.js";import"./echarts.7885de64.js";import"./tslib.60310f1a.js";import"./zrender.6135380a.js";import"./lodash-es.a286bec7.js";import"./@vueuse.8dbaa908.js";import"./@popperjs.36402333.js";import"./@ctrl.82a509e0.js";import"./dayjs.46f97078.js";import"./async-validator.fb49d0f5.js";import"./memoize-one.4ee5c96d.js";import"./escape-html.e5dfadb9.js";import"./normalize-wheel-es.8aeb3683.js";import"./@floating-ui.f14f28ca.js";const M="/assets/WoRMS.e01cab11.png",I="/assets/NCBI-Taxonomy.e3b461e6.png",N="/assets/MolluskDB.71a75958.jpg",T="/assets/BOLDSYSTEMS.83561ecb.png";const C={components:{AppHeader:y,AppFooter:v},data(){return{visible:!1,email:""}},mounted(){w({title:"Friends websites",message:"Friends websites, database and servers!",type:"success",duration:3e3,position:"bottom-left"})}},t=l=>(S("data-v-d30ebef4"),l=l(),D(),l),R=t(()=>e("br",null,null,-1)),j=t(()=>e("br",null,null,-1)),k=t(()=>e("br",null,null,-1)),A=t(()=>e("h2",null,"Taxonomy Databases:",-1)),E={style:{width:"100%"}},W={style:{"font-size":"1.2em","line-height":"1.5em","text-align":"justify"}},F=t(()=>e("h1",{class:"hightlight-blue"}," WoRMS: World Register of Marine Species ",-1)),L=t(()=>e("p",null,[e("span",{class:"hightlight-blue"}," World Register of Marine Species (WoRMS) "),n(" The aim of a World Register of Marine Species (WoRMS) is to provide an authoritative and comprehensive list of names of marine organisms, including information on synonymy. While the highest priority goes to valid names, other names in use are included so that this register can serve as a guide to interpret taxonomic literature. "),e("span",{class:"hightlight-blue"}," The content of WoRMS is controlled by taxonomic and thematic experts, not by database managers. "),n(" WoRMS has an editorial management system where each taxonomic group is represented by an expert who has the authority over the content, and is responsible for controlling the quality of the information. Each of these main taxonomic editors can invite several specialists of smaller groups within their area of responsibility to join them. "),e("span",{class:"hightlight-blue"}," This register of marine species grew out of the European Register of Marine Species (ERMS), "),n(" and its combination with several other species registers maintained at the Flanders Marine Institute (VLIZ). ")],-1)),z=t(()=>e("img",{src:M,style:{width:"100%","aspect-ratio":"true","object-fit":"cover","border-radius":"10px","box-shadow":"0px 0px 20px #eeeeee"}},null,-1)),q=t(()=>e("br",null,null,-1)),O=t(()=>e("br",null,null,-1)),V=t(()=>e("br",null,null,-1)),H={style:{width:"100%"}},$=t(()=>e("img",{src:I,style:{width:"100%","aspect-ratio":"true","object-fit":"cover","border-radius":"10px","box-shadow":"0px 0px 20px #eeeeee"}},null,-1)),Y={style:{"font-size":"1.2em","line-height":"1.5em","text-align":"justify"}},P=t(()=>e("h1",{class:"hightlight-green"}," NCBI Taxonomy ",-1)),X=t(()=>e("br",null,null,-1)),Z=t(()=>e("br",null,null,-1)),G=t(()=>e("span",{class:"hightlight-green"}," The National Center for Biotechnology Information (NCBI) Taxonomy ",-1)),J=t(()=>e("span",{class:"hightlight-green"}," An early version of the NCBI Taxonomy in 1991 ",-1)),K=t(()=>e("span",{class:"hightlight-green"}," In 1996 the first version of NCBI Taxonomy Web Browser ",-1)),Q=t(()=>e("br",null,null,-1)),U=t(()=>e("br",null,null,-1)),ee=t(()=>e("br",null,null,-1)),te={style:{width:"100%"}},se={style:{"font-size":"1.2em","line-height":"1.5em","text-align":"justify"}},oe=t(()=>e("h1",{class:"hightlight-blue"}," MolluscDB: Molluscan genomics Database ",-1)),ne=t(()=>e("p",null,[e("span",{class:"hightlight-blue"}," An integrated functional and evolutionary genomics database for the hyper-diverse animal phylum Mollusca "),n(" Mollusca represents the second largest animal phylum but remains poorly explored from a genomic perspective. "),e("span",{class:"hightlight-blue"}," MolluskDB compiles and integrates current molluscan genomic/transcriptomic resources and provides convenient tools for multi-level integrative and comparative genomic analyses. "),n(" MolluscDB enables a systematic view of genomic information from various aspects, such as genome assembly statistics, genome phylogenies, fossil records, gene information, expression profiles, etc. "),e("span",{class:"hightlight-blue"}," MolluscDB offers valuable customized datasets or resources, "),n(" such as gene coexpression networks across various developmental stages and adult tissues/organs, core gene repertoires inferred for major molluscan lineages, and macrosynteny analysis for chromosomal evolution. ")],-1)),ie=t(()=>e("img",{src:N,style:{width:"100%","aspect-ratio":"true","object-fit":"cover","border-radius":"10px","box-shadow":"0px 0px 20px #eeeeee"}},null,-1)),ae=t(()=>e("br",null,null,-1)),le=t(()=>e("br",null,null,-1)),re=t(()=>e("br",null,null,-1)),ce={style:{width:"100%"}},he=t(()=>e("img",{src:T,style:{width:"100%","aspect-ratio":"true","object-fit":"cover","border-radius":"10px","box-shadow":"0px 0px 20px #eeeeee"}},null,-1)),de={style:{"font-size":"1.2em","line-height":"1.5em","text-align":"justify"}},pe=t(()=>e("h1",{class:"hightlight-green"}," BOLD SYSTEMS: Barcode of Life Data Systems ",-1)),_e=t(()=>e("br",null,null,-1)),ue=t(()=>e("br",null,null,-1)),me=t(()=>e("span",{class:"hightlight-green"}," The Barcode of Life Data Systems (BOLD) ",-1)),ge=t(()=>e("span",{class:"hightlight-green"}," By assembling molecular, morphological, and distributional data, it bridges a traditional bioinformatics chasm. ",-1)),fe=t(()=>e("span",{class:"hightlight-green"}," Because of its web-based delivery and flexible data security model, it is also well positioned to support projects that involve broad research alliances. ",-1)),be=t(()=>e("br",null,null,-1)),ye=t(()=>e("br",null,null,-1)),ve=t(()=>e("br",null,null,-1));function we(l,xe,Be,Se,De,Me){const h=i("AppHeader"),d=i("el-header"),p=i("el-divider"),r=i("el-link"),a=i("el-col"),c=i("el-row"),_=i("el-main"),u=i("AppFooter"),m=i("el-footer"),g=i("el-container"),f=i("el-backtop");return x(),B("div",null,[s(g,null,{default:o(()=>[s(d,null,{default:o(()=>[s(h)]),_:1}),s(_,null,{default:o(()=>[R,j,k,A,s(p,{"content-position":"left"},{default:o(()=>[n("Featured taxonomy databases")]),_:1}),e("div",E,[s(c,{gutter:50},{default:o(()=>[s(a,{span:24,sm:12,lg:12},{default:o(()=>[e("div",W,[F,s(r,{href:"https://www.marinespecies.org/index.php",target:"_blank",class:"hightlight-blue"},{default:o(()=>[n(" https://www.marinespecies.org/ ")]),_:1}),L])]),_:1}),s(a,{span:24,sm:12,lg:12},{default:o(()=>[z]),_:1})]),_:1})]),q,O,V,e("div",H,[s(c,{gutter:50},{default:o(()=>[s(a,{span:24,sm:12,lg:12},{default:o(()=>[$]),_:1}),s(a,{span:24,sm:12,lg:12},{default:o(()=>[e("div",Y,[P,s(r,{href:"https://www.ncbi.nlm.nih.gov/taxonomy",class:"hightlight-green"},{default:o(()=>[n(" https://www.ncbi.nlm.nih.gov/taxonomy ")]),_:1}),X,Z,G,n(" includes organism names and classifications for every sequence in the nucleotide and protein sequence databases of the International Nucleotide Sequence Database Collaboration (INSDC). "),J,n(" was included in the first version of Entrez, the search and retrieval system for NCBI\u2019s linked databases. "),K,n(" was presented to the public and the INSDC decided on its annual meeting to utilize the NCBI Taxonomy as the only source for taxonomic classification to maintain consistency among databases. ")])]),_:1})]),_:1})]),Q,U,ee,e("div",te,[s(c,{gutter:50},{default:o(()=>[s(a,{span:24,sm:12,lg:12},{default:o(()=>[e("div",se,[oe,s(r,{href:"https://www.marinespecies.org/index.php",target:"_blank",class:"hightlight-blue"},{default:o(()=>[n(" http://mgbase.qnlm.ac ")]),_:1}),ne])]),_:1}),s(a,{span:24,sm:12,lg:12},{default:o(()=>[ie]),_:1})]),_:1})]),ae,le,re,e("div",ce,[s(c,{gutter:50},{default:o(()=>[s(a,{span:24,sm:12,lg:12},{default:o(()=>[he]),_:1}),s(a,{span:24,sm:12,lg:12},{default:o(()=>[e("div",de,[pe,s(r,{href:"https://www.ncbi.nlm.nih.gov/taxonomy",class:"hightlight-green"},{default:o(()=>[n(" https://www.boldsystems.org ")]),_:1}),_e,ue,me,n(" is an informatics workbench aiding the acquisition, storage, analysis, and publication of DNA barcode records. "),ge,n(" BOLD is freely available to any researcher with interests in DNA barcoding. By providing specialized services, it aids the assembly of records that meet the standards needed to gain BARCODE designation in the global sequence databases. "),fe,n(" Ratnasingham, S. & Hebert, P. D. N. (2007). BOLD: The Barcode of Life Data System (www.barcodinglife.org). Molecular Ecology Notes 7, 355-364. DOI: 10.1111/j.1471-8286.2006.01678.x ")])]),_:1})]),_:1})]),be,ye,ve]),_:1}),s(m,null,{default:o(()=>[s(u)]),_:1})]),_:1}),s(f,{right:50,bottom:100})])}const Ge=b(C,[["render",we],["__scopeId","data-v-d30ebef4"]]);export{Ge as default};
